app/controllers/listings_controller.rb

```diff
@@ -57,6 +59,10 @@ def destroy
     redirect_to properties_url
   end
 
+  def delete_image
+    
+  end
+
```

Comment:
Please complete the delete method on this task.

Reply:
Delete is already working and its in the images_controller I forgot to clear this one out during testing